packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts

```diff
@@ -30,4 +30,22 @@ export interface TelemetryEventMessage extends IDebuggerMessage<TelemetryEventMe
 	type: "TELEMETRY_EVENT";
 }
 
+/**
+ * Outbound event indicating that all telemetry events has been generated by the application.
```

Comment:
Nit: This message is posted in response to a request, so I don't think its being posted really "indicates" anything. I might say something like:
```suggestion
 * Outbound message including the entire history of telemetry events.
```